examples/todolist/index.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable */
+
+import express from 'express';
+import bodyParser from 'body-parser';
+import pug from 'pug';
+import uuid from 'uuid';
+import Immutable from 'seamless-immutable';
+
+import createStore from 'resolve-es';
+import esDriver from 'resolve-es-file';
+import createBus from 'resolve-bus';
+import busDriver from 'resolve-bus-memory';
+import commandHandler from 'resolve-command';
+import query from 'resolve-query';
+
+import todoCardAggregate from './aggregates/TodoCard';
+import todoItemAggregate from './aggregates/TodoItem';
+import cardsReadModel from './read-models/cards';
+import cardDetailsReadModel from './read-models/cardDetails';
+
+const setupMiddlewares = (app) => {
+    app.use( bodyParser.json() );
```

Comment:
Where is it used? (*/:card* is the only candidate and looks like it does not use)